roles/multi-tenancy/tasks/multi-tenant-onboard-offboard.yaml

```diff
@@ -52,34 +52,62 @@
   tags:
     - offboard
 
-# Build the manifest file
+- name: kustomize - get load restrictor flag
+  set_fact:
+      LOAD_RESTRICTOR_FLAG: "{{ '--load-restrictor LoadRestrictionsNone' if V4_CFG_CADENCE_VERSION is version('2023.02', '>=') or V4_CFG_CADENCE_NAME|lower == 'fast' else '--load_restrictor=none' }}"
+  tags:
+    - onboard
+    - offboard
+
 - name: kustomize - generate onboard manifest
   block:
-    - name: kustomize - generate onboard manifest
-      ansible.builtin.shell: |
-        kustomize build {{ DEPLOY_DIR }}/site-config/sas-tenant-job/ --load-restrictor LoadRestrictionsNone -o {{ DEPLOY_DIR }}/site-config/sas-tenant-job/site-onboard.yaml
-      when: V4_CFG_CADENCE_VERSION is version('2023.02', ">=") or V4_CFG_CADENCE_NAME|lower == "fast"
-    - name: kustomize - generate onboard manifest - legacy flags
-      ansible.builtin.shell: |
-        kustomize build {{ DEPLOY_DIR }}/site-config/sas-tenant-job/ --load_restrictor='none' -o {{ DEPLOY_DIR }}/site-config/sas-tenant-job/site-onboard.yaml
+    - name: kustomize - generate onboard manifest - Ansible
+      ansible.builtin.shell: >
```

Comment:
What's the purpose of the `>` directive vs the pipe `|` directive?

Reply:
I'm using the ">" directive (Folded Block Scalar) so that new lines in the middle of my multiline block are interpreted as spaces and not newlines. If the pipe was used every line where I split up my multiline block would be interpreted as a newlines.

A quick example
```
- name: my task
  ansible.builtin.shell: >
	kubectl get
	pods -n mynamespace
```
would be run as `kubectl get pods -n mynamespace\n`

```
- name: my task
  ansible.builtin.shell: |
	kubectl get
	pods -n mynamespace
```

would be run as `kubectl get\npods -n mynamespace\n`. The newline that is retained in the middle of the command would be problematic when using the pipe.

No direct link to it in the documentation, but search for the string "Folded Block Scalar" on the Ansible doc page. Which has a good explanation 
https://docs.ansible.com/ansible/latest/reference_appendices/YAMLSyntax.html#yaml-basics